include/psa/crypto_extra.h

```diff
@@ -2069,6 +2069,264 @@ static inline struct psa_pake_operation_s psa_pake_operation_init(void)
+ *       OneAsymmetricKeyAttributes ATTRIBUTE ::= {
+ *         ...   -- For local profiles
+ *       }
+ *
+ *  Key attributes when importing:
+ *  . key permitted-algorithm: required in the case of rsaEncryption and
+ *    id-ecPublicKey algorithm identifiers for keys that will be used for a
+ *    cryptographic operation. Otherwise, if defined, the import functions
+ *    check that it is compatible with the algorithm specified in key data.
+ *  . key usage: a key usage may be provided to limit the usage of the key to
+ *    encryption/decryption or signature/verification only according to its
+ *    permitted-algorithm.
+ */
+    PSA_KEY_DATA_FORMAT_ONE_ASYMMETRIC_KEY,
+
+    PSA_KEY_DATA_FORMAT_COUNT
+} psa_key_data_format_t;
+
+/**
+ * \brief Import a key in one of the supported key data formats.
+ *
+ * This functions supports the formats as defined in the documentation of
+ * #psa_key_data_format_t.
+ *
+ * \param[in] attributes    The attributes for the new key. This function uses
+ *                          the attributes as follows:
+ *                          . The key type is inferred from the key data
+ *                            in \p data or the key data format \p format (see
+ *                            the documentation of #psa_key_data_format_t for
+ *                            more information). If provided in \p attributes,
+ *                            it must match the inferred key type.
+ *                          . The key size is always determined from the key
+ *                            data. If the key size in \p attributes is nonzero,
+ *                            it must be equal to the size determined from the
+ *                            key data.
+ *                          . The key permitted-algorithm policy may be
+ *                            required for keys that will be used for a
+ *                            cryptographic operation, see the documentation of
+ *                            the key data formats for more information. If the
+ *                            key data imposes restrictions on the algorithms
+ *                            the key can be used for, the key
+ *                            permitted-algorithm policy must comply with
+ *                            those restrictions.
+ *                          . The key usage flags define what operations are
+ *                            permitted with the key. If the key data imposes
+ *                            restrictions on the operations the key can be
+ *                            used for, the key usage flags must comply with
+ *                            those restrictions.
+ *                          . The key lifetime and identifier are required
+ *                            for a persistent key.
+ *
+ * \param[in] format        The format of the key data in the \p data buffer.
+ * \param[out] key          On success, an identifier to the newly created key.
```

Comment:
Move this parameter to the end of the definition list, to match the function declaration